bridge/src/commonMain/kotlin/ru/herobrine1st/matrix/bridge/telegram/TelegramWorker.kt

```diff
@@ -36,7 +38,23 @@ class TelegramWorker(
         actorId: TelegramActorId,
         id: UserId
     ): RemoteUser<UserId> {
-        TODO("Not yet implemented")
+        val actorData: TelegramActorData? = actorProvisionRepository.getActorData(actorId)
+        val token = actorData?.token ?: throw IllegalStateException("No token for actor $actorId")
```

Comment:
```suggestion
        val actorData = actorProvisionRepository.getActorData(actorId)
        val token = actorData.token
```
`getActorData` already throws exception if actor is not found